snap/snaplock/lock.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package mount
+package snaplock
```

Comment:
this could be have been its own PR,  it's  a bit unclear if it belongs under snap (not that I we have a clear better place) ... is this lock intrinsic to snaps as a format and how they are meant to be delivered?  (the line is blurry but most of the content of snap/ follows that line)

Reply:
I could document the meaning of the lock. Not sure if you have a concrete proposal on how to move forward.

Reply:
given is used by snap-update-ns  and is the same lock as snap-confine,  I would move it to cmd/snaplock for now (it's not ideal but we have some other things we use out of cmd that are a bit odd)

Reply:
Hmm, I find that very odd, cmd has no non-command packages. Why is that better in your eyes than `snap/snaplock`? As we work on this feature I expect we will need another type of lock that would also make sense to add to `snaplock` (the one with inhibit semantics that can be held longer).

Reply:
we already have cmd/snapinfo.go  with he same kind of problem (we need a place for that and this but for now they can go there),  as I said I really prefer to maintain  some distinction between more intrinsic aspect of snaps,  and more implementation details for them

Reply:
Are you proposing that `snaplock` then moves to `cmd/snaplock.go`?

Reply:
no, I'm still asking for a cmd/snaplock package

Reply:
if it sounds too much like a non-existing command, snaplocking is also a fine name

Reply:
Also it doesn't seem to have a comment about the fact that this kind of lock is created also from snap-confine, a pointer to the corresponding c code.